src/components/common/input/components/default-input.js

```diff
@@ -1,43 +1,43 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import styles from '../input.module.scss';
+// import React from 'react';
```

Comment:
i will be removing this file completely but it's tied to many places and it could lead to different issues.